pkg/keb/contract.go

```diff
@@ -19,17 +19,17 @@ type Configuration struct {
 	Secret bool        `json:"secret"`
 }
 
-type Components struct {
+type Component struct {
 	Component     string          `json:"component"`
```

Comment:
Does it make sense to rename this as `Name`? What do you think?

Reply:
Then it'd be `Component.Name` instead of `Component.Component`.

Reply:
Agree, that would be better.. will rename it